samples/snippets/snippets_mute_config.py

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/env python
+#
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START securitycenter_create_mute_config]
+from google.cloud.securitycenter_v1 import Finding
+
+
+def create_mute_rule(parent_path: str, mute_config_id: str) -> None:
+    """
+    Creates a mute configuration under a given scope that will mute
+    all new finding which match the filter/ mute rule.
+    Existing findings will NOT BE muted.
+    Args:
+        parent_path: use any one of the following three options,
+                    organizations/{organization} or
```

Comment:
Nit: Since you already state that they should only use one of the options, how about adding dashes so they appear as bullet points, and then deleting the "or". 

- organizations/{organization_id}
- folders/{folder_id}
- projects/{project_id}

It might also be worth noting that these are all numeric IDs.

Reply:
Thanks Catherine, mentioning it as numeric id will remove the datatype confusion.